app/models/user.rb

```diff
@@ -13,6 +13,7 @@ class User < ActiveRecord::Base
 
   acts_as_authentic do |c|
     c.openid_required_fields = %i(nickname email)
+    VALID_EMAIL_REGEX = /\A[\w+-.]+@[a-z\d-.]+.[a-z]+\z/i
```

Comment:
Please add the regex expression into 
`` c.validates_format_of_email_field_options = { with: /@/ }``
also we had a discussion about regex in other pr, please have a look at it. 
Please refer rubular for regex.